script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -1,29 +1,29 @@
 <Page1>
-The rail line above ground switches to an underground line without a hint of warning.
-The train, spewing artificial light, moves through the darkness as if it were diving into it.
+The rail line above ground switches to an underground line without warning.
+The train, spewing artificial light, dives into the darkness.
 <Page2>
 The sound of creaking vehicles.
 Artificial lights passing by at regular intervals.
 As the tremors of the seat overlap with the second hand of the clock, I think about the passage of time and space.
 <Page3>
 It's early morning, 6:33 am.
 Half an hour has passed since I got on this train.
-I can't go back anymore, all this time and distance is now keeping me apart from the family that took care of me for all these years.
+I can't go back anymore. All this time and distance is now keeping me apart from the family that took care of me for all these years.
 How I accept this may be a measure of the depth of my compassion as a human being.
 <Page4>
-Do you think of them as being 'still that close'?
-Or write them off as being 'already so far away'?
+Despite our physical distance, do I think of them as still being close? // still to be rechecked
+Or have I written them off as part of the past?
 <Page5>
-I wondered which of the two I feel.
-I thought about it for a moment and then realized I didn't feel either of those things.
+I wonder which of the two I feel.
+I think about it for a moment, and then realize I don't feel either.
 <Page6>
-I am the only one on the eight-person bench, and the row facing me is entirely empty.
+I'm the only one on the eight-person bench, and the row facing me is entirely empty.
 The only passengers are a man in a suit dozing off in the next row over, and a girl standing in front of the door.
-Because it is such an early morning train, it is very lonely inside the train car.
+Because it's such an early morning route, it's very lonely inside the train car.
```

Comment:
Revert route -> train. The train is early, the route the train takes doesn't change depending on what time it is